frontend/public/components/secrets/create-secret/OpaqueSecretFormEntry.tsx

```diff
@@ -20,48 +25,49 @@ export const OpaqueSecretFormEntry: React.FC<OpaqueSecretFormEntryProps> = ({
     onChange(updatedEntry, index);
   };
 
-  const handleKeyChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+  const handleKeyChange = (_event: FormEvent<HTMLInputElement>, value: string) => {
     onChange(
       {
         ...entry,
-        key: event.target.value,
+        key: value,
       },
       index,
     );
   };
 
   return (
-    <div className="co-create-generic-secret__form">
-      <div className="form-group">
-        <label className="co-required" htmlFor={`${index}-key`}>
-          {t('public~Key')}
-        </label>
-        <div>
-          <span className="pf-v6-c-form-control">
-            <input
-              id={`${index}-key`}
-              type="text"
-              name="key"
-              onChange={handleKeyChange}
-              value={entry.key}
-              data-test="secret-key"
-              required
-            />
-          </span>
-        </div>
-      </div>
-      <div className="form-group">
-        <DroppableFileInput
-          onChange={handleValueChange}
-          inputFileData={Base64.decode(entry.value)}
-          id={`${index}-value`}
-          label={t('public~Value')}
-          inputFieldHelpText={t(
-            'public~Drag and drop file with your value here or browse to upload it.',
-          )}
-          inputFileIsBinary={entry.isBinary_}
+    <FormFieldGroup>
```

Comment:
I found a way to mangle PatternFly to remove the indentation we didn't want

```suggestion
    <FormFieldGroup className="pf-v6-u-display-block">
```